subscription-engine/kafka-ws/config/default.conf

```diff
@@ -0,0 +1,38 @@
+# LOGGER
+logger.transports.console.level=info
+logger.verbose:boolean=false
+
+logger.file.enable:boolean=false
+logger.file.level=info
+logger.file.filename=kafka-ws-logs-%DATE%.log
```

Comment:
I would change the names to:
```
logger.transports.file.*
```
This way we keep the same pattern as the console configuration.